cfy_manager/utils/certificates.py

```diff
@@ -581,3 +592,14 @@ def get_cert_and_key_filenames(new_cert_location,
 def get_ca_filename(new_ca_location, default_ca_location):
     return (new_ca_location if os.path.exists(new_ca_location)
             else default_ca_location)
+
+
+def _get_md5_hash(cert_path):
+    content = sudo(['openssl', 'x509', '-noout', '-modulus', '-in', cert_path])
+    md5_res = sudo(['openssl', 'md5'], stdin=content.aggr_stdout)
+    md5_hash = md5_res.aggr_stdout.split('(stdin)= ')[1]
+    return md5_hash
+
+
+def two_certs_identical(cert_a, cert_b):
```

Comment:
Maybe just call this certs_identical, no need to state that it's 'two'?

Reply:
Yep, makes sense, changing.